server/mailet/integration-testing/src/test/java/org/apache/james/mailets/SubAddressingTest.java

```diff
@@ -196,6 +203,28 @@ void subAddressedEmailShouldBeDeliveredInINBOXWhenSpecifiedFolderExistsWithCorre
         awaitSubAddressedMail(MailboxConstants.INBOX);
     }
 
+    @Disabled("can't manage to successfully create the mailbox...")
+    @Test
+    void subAddressedEmailShouldBeDeliveredInSpecifiedFolderWhenRequiringEncoding(@TempDir File temporaryFolder) throws Exception {
+        setup(temporaryFolder);
+
```

Comment:
```suggestion
jamesServer.probe(MailboxProbeImpl.class).createMailbox(MailboxPath.forUser(RECIPIENT, TARGETED_MAILBOX_REQUIRING_ENCODING));
```

Reply:
(let's cheat for that one, utf -7 is tricky)

Reply:
right, thank you !